forest/src/cli/client.rs

```diff
@@ -23,6 +23,7 @@ pub struct Client {
     pub snapshot: bool,
     pub snapshot_height: Option<i64>,
     pub snapshot_path: Option<String>,
+    pub exit_after_import: bool,
```

Comment:
Would be great to use the same wording as in Lotus which is `--halt-after-import`.

https://lotus.filecoin.io/lotus/manage/chain-management/#restoring-a-custom-snapshot

Most of Forest cli args are already similar to theirs.

Reply:
Oh, I didn't know this was a thing. I'll rename it